lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -438,11 +437,7 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            if host[:type] == 'aio'
```

Comment:
Yeah, I think that's fine as long as we have a way to install the older stuff as well.